app/src/main/java/com/example/concordia_campus_guide/Helper/DrawDirectionsPolyLines.java

```diff
@@ -0,0 +1,44 @@
+package com.example.concordia_campus_guide.Helper;
+
+import android.graphics.Color;
+
+import com.example.concordia_campus_guide.Adapters.DirectionWrapper;
+import com.example.concordia_campus_guide.ClassConstants;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.PolylineOptions;
+
+import java.util.List;
+
+public class DrawDirectionsPolyLines {
+
+    public void drawOutdoorPath(List<DirectionWrapper> outdoorDirections, GoogleMap map) {
+        for (DirectionWrapper directionWrapper : outdoorDirections) {
+            int color = 0;
+            if (directionWrapper.getTransitDetails() != null) {
+                color = Color.parseColor(directionWrapper.getTransitDetails().line.color);
+            }
+            PolylineOptions polylineOptions = stylePolyLine(directionWrapper.getDirection().getTransportType(), color);
+            List<com.example.concordia_campus_guide.GoogleMapsServicesTools.GoogleMapsServicesModels.LatLng> polyline = directionWrapper.getPolyline().decodePath();
```

Comment:
if it is a list would it make more sense to name it polylines? also it does not change much but it would be sexier to put for ( polyline : polylines) 
polylineOptions.add(new LatLng(polyline.lat, polyline.lng));

Reply:
hahaha good catch 👍  and done 👍